util/src/test/java/tech/pegasys/artemis/util/resource/URLResourceLoaderTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.util.resource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.net.URL;
+import org.junit.jupiter.api.Test;
+
+class URLResourceLoaderTest {
+  private final ResourceLoader loader = new URLResourceLoader();
+
+  @Test
+  public void shouldLoadContentFromURL() throws Exception {
+    final String resourceName = ClasspathResourceLoaderTest.TEST_FILE_1 + ".txt";
+    final URL resource = ClasspathResourceLoaderTest.class.getResource(resourceName);
```

Comment:
It's not clear to me how you're testing loading from URL here

Reply:
It's cheating a little - I'm getting a URL to a resource on the class path.  That's likely actually either a `file://` or `jar://` URL but since the implementation is just using the standard `URL.openStream` any URL supported by that works well enough to provide a basic sanity test.  Previously this loading from URL wasn't tested at all.